app/models/draft_consent.rb

```diff
@@ -105,7 +108,15 @@ def wizard_steps
   end
 
   on_wizard_step :agree, exact: true do
-    validates :response, inclusion: { in: Consent.responses.keys }
+    validates :response,
+              inclusion: {
+                in: Consent.responses.keys + FLU_RESPONSES
+              }
```

Comment:
Would it make sense to make this validation depend on the programme type? Something like:

```suggestion
    validates :response,
              inclusion: { in: Consent.responses.keys },
              unless: -> { programme.flu? }
    validates :response,
              inclusion: { in: FLU_RESPONSES },
              unless: -> { programme.flu? }
```